Formula/dart@1.rb

```diff
@@ -4,7 +4,6 @@
 class DartAT1 < Formula
   desc "Legacy Dart 1 SDK"
   homepage "https://dart.dev"
-  version "1.24.3"
```

Comment:
so version is inferred?

Reply:
Yes, otherwise audit complains:
```
dart@1:
  * Stable: version 1.24.3 is redundant with version scanned from URL
  * Stable resource "content_shell": version 1.24.3 is redundant with version scanned from URL
  * Stable resource "dartium": version 1.24.3 is redundant with version scanned from URL
```